manifests/config.pp

```diff
@@ -64,12 +72,6 @@
         notify  => Exec['systemctl-daemon-reload'],
         require => Class['caddy::package'],
       }
-
-      exec {'systemctl-daemon-reload':
```

Comment:
the exec also needs to come back, the file resource references it in line 72.

Reply:
missed that, fixed.